### PR TITLE
Encapsulate Token operations

### PR DESCRIPTION
**Pull request by WillardHu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Encapsulate Token operations to improve readability and maintainability.

**Which issue(s) this PR fixes**:

A part of issue #5498 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
Is this ready for review? :) @WillardHu

**Comment by WillardHu** [comment]
> Is this ready for review? :) @WillardHu

Yes, this task has been completed.

**Comment by WillardHu** [comment]
This change passed some tests in my k8s cluster
- Cloudcore generates a self-signed ca certificate initially ✅
- An new joined edge node(Edgecore) can connect to Cloudcore and obtains a service certificate ✅
- The token can be correctly verified when the edge node request to the Cloudcore with a Authentication header ✅

PTAL  @fisherxu @Shelley-BaoYue

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 1]
```diff
@@ -141,7 +142,9 @@ func verifyCertSubject(cert *x509.Certificate, nodeName string) error {
 // verifyAuthorization verifies the token from EdgeCore CSR
 func verifyAuthorization(w http.ResponseWriter, r *http.Request) bool {
 	authorizationHeader := r.Header.Get("authorization")
+	klog.V(4).Info("authorization token is: ", authorizationHeader)
```
[the remark this answers is not in the document]
Comment (WillardHu) [reply cloud/pkg/cloudhub/servers/httpserver/server.go]: The log print has been deleted, thanks

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @Shelley-BaoYue @luomengY

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Pull request merged**